### ISSUE TITLE
set pyramid_tm to use the explicit transaction manager in the cookiecutters

### ISSUE THREAD

**Issue by mmerickel** [Pylons/pyramid#3052]
pyramid_tm supports and recommends using an explicit transaction transaction manager for apps so that they do not implicitly create new transactions when touching the manager outside of the pyramid_tm lifecycle. We should update the cookiecutters to use this new manager. The fix is simple, aside from verifying that the docs / tutorials are up to date:

```
settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
```

**Comment by stevepiercy** [comment]
This issue will also require an update to the source files in the docs.

A quick search for `config = Configurator(root_factory=root_factory, settings=settings)` returns 6 matches in 6 files in the docs/ directory.

**Comment by bertjwregeer** [comment]
This is being worked on in: https://github.com/Pylons/pyramid/pull/3059

**Comment by stevepiercy** [comment]
Remaining work is now listed in #3081.

**Comment by mmerickel** [comment]
Fixed in #3082 and related cookiecutter issues.
